devel/xcode-build-server/Portfile

```diff
@@ -4,10 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        SolaWing xcode-build-server 1.1.0 v
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
-version             1.1.0
+github.setup        SolaWing xcode-build-server 1.2.0 v
+github.tarball_from archive
+version             1.2.0
```

Comment:
the `version` field is set automatically from the `github.setup` line so please remove

Reply:
Done